src/sentry/migrations/monkey/writer.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+# This should be exactly the same as `django.db.migrations.writer.py.MIGRATION_TEMPLATE`,
+# except that we add
+# - `atomic = False`
+# - `is_dangerous = False`
+# to the class definition. Compare this template after each Django version bump to make
+# sure we're not missing any important changes.
+SENTRY_MIGRATION_TEMPLATE = """\
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+%(imports)s
+
+class Migration(migrations.Migration):
+    # We prefer to run most database schema changes, and all data migrations outside of
+    # a transaction. If you need your ddl to run inside of a transaction feel free to
+    # change this to True, but in most cases this should be unnecessary.
```

Comment:
You might wanna separate this comment away from the `is_dangerous` flag. I also got sorta confused reading this. But it also is probably worth explaining that flag too in a block comment.